index.html

```diff
@@ -0,0 +1,214 @@
+      href="https://fonts.googleapis.com/css2?family=Harmattan&family=Inter:wght@200&family=Roboto+Serif:wght@100&family=Roboto:ital,wght@1,100&display=swap"
+      rel="stylesheet"
+    />
+
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Harmattan&family=Roboto+Serif:wght@100&family=Roboto:ital,wght@1,100&display=swap"
+      rel="stylesheet"
+    />
+    <title>Gideon's Portfolio</title>
+    <script
+      src="https://kit.fontawesome.com/9947fbd823.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+
+  <body>
+    <header>
+      <a href="#" class="name_logo">Gidiz Logo</a>
+      <nav>
+        <img class="menu-bar" src="icons_images/Union.png" alt="Menu" />
+      </nav>
+    </header>
+
+    <section class="main-mobile-section">
+      <div class="ellipse1"></div>
+      <div class="ellipse2"></div>
+
+      <h1>Gideon Dadi</h1>
+      <h3>Software Developer</h3>
+      <div class="advert-text">
+        <p>
+          I can help you build a product, feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
+
+      <hr class="vector4" />
+
+      <div class="tech-experience">
+        <ul class="language">
+          <li><h2>Language</h2></li>
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>CSS</li>
+          <li>Html</li>
+        </ul>
+
+        <ul class="frameworks">
+          <li><h2>Frameworks</h2></li>
+          <li>Bootstrap</li>
+          <li>Ruby on Rails</li>
+          <li>RSPec</li>
+          <li>CapyBara</li>
+          <li>Selenium</li>
+        </ul>
+
+        <ul class="skills">
+          <li><h2>Skills</h2></li>
+          <li>CodeKit</li>
+          <li>GitHub</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
+
+        <hr class="vector5" />
+      </div>
+
+      <ul class="social-media-icons">
+        <li>
+          <a href="#">
+            <img src="icons_images/Vector.png" alt="" />
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="icons_images/Vector4.png" alt="" />
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="icons_images/Vector3.png" alt="" />
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="icons_images/Vector2.png" alt="" />
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="icons_images/Vector5.png" alt="" />
+          </a>
+        </li>
+      </ul>
+
+      <div class="work-arrow">
+        <a href="#"> <img src="icons_images/Vector6.png" alt="" /> Work </a>
+      </div>
+
+      <h1 class="my-work-header">MY RECENT WORK</h1>
```

Comment:
- [ ] Please change this h1 to hx grade below, as for HTML best practice, your web app should only have one h1 tag, consider doing this for all similar titles.

Reply:
Awesome! Thanks for the correction